domino_cost/cost.py

```diff
@@ -88,24 +105,31 @@ def clean_df(df: DataFrame, col: str) -> DataFrame:
     return df[~df[col].str.startswith("__")]
 
 
-def get_cumulative_cost_graph(cost_table: DataFrame, time_span: timedelta) -> dict:
+def get_cumulative_cost_graph(cost_table: DataFrame, time_span: str) -> dict:
+    start, end = time_span.split(",")
     x_date_series = pd.date_range(
-        get_today_timestamp() - get_time_delta(time_span),
-        get_today_timestamp(),
+        to_pd_ts(start),
+        to_pd_ts(end),
     ).strftime("%B %-d")
     cost_table_grouped_by_date = cost_table.groupby("FORMATTED START")
 
-    colors = ['#FF6543','#A5C5F6', '#563FB3', '#3C3A42', '#777384',]  # Color codes
+    colors = [
```

Comment:
I see 5 colors being defined, but only 3 are being used (looking at `marker=dict(color=colors[i])`)

Reply:
The other two will be used later on, so we just want to keep them in there instead of redefining them later.